lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -166,17 +166,23 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     }
 
     // Budgets
-    const budgetAudit = category.auditRefs.find(audit => audit.id === 'performance-budget');
-    if (budgetAudit && budgetAudit.result.details) {
-      const table = this.detailsRenderer.render(budgetAudit.result.details);
-      if (table) {
-        table.id = budgetAudit.id;
-        table.classList.add('lh-audit');
-        const budgetsGroupEl = this.renderAuditGroup(groups.budgets);
-        budgetsGroupEl.appendChild(table);
-        budgetsGroupEl.classList.add('lh-audit-group--budgets');
-        element.appendChild(budgetsGroupEl);
+    const budgetsGroupEl = this.renderAuditGroup(groups.budgets);
+    let showBudgetsGroup;
```

Comment:
might be better to have a `budgetTableEls` array, push to it ~L176 (instead of append to `budgetsGroupEl`), and check it's non-empty before creating `budgetsGroupEl`.